src/main/java/de/unistuttgart/iste/gits/skilllevel_service/service/ContentServiceClient.java

```diff
@@ -66,69 +64,107 @@ public List<Content> getContentsWithUserProgressData(UUID userId,
             String query = """
                     query($userId: UUID!, $chapterIds: [UUID!]!) {
                         contentsByChapterIds(chapterIds: $chapterIds) {
-                            id
-                            metadata {
-                                name
-                                tagNames
-                                suggestedDate
-                                type
-                                chapterId
-                                rewardPoints
-                            }
-                            progressDataForUser(userId: $userId) {
-                                userId
-                                contentId
-                                learningInterval
-                                nextLearnDate
-                                lastLearnDate
-                                log {
-                                    timestamp
-                                    success
-                                    correctness
-                                    hintsUsed
-                                    timeToComplete
+                            ...on Assessment {
```

Comment:
What happens with media content? then it will not return anything or how does it work?

Reply:
my solution actually does not work. I thought I tested it with media contents in the chapter but I didn't.